operator/secret.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
+  password: ZmFsa29yZGI=
+kind: Secret
+metadata:
+  name: redis-password
+  namespace: {{$sys.id}}
+type: Opaque
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)